index.js

```diff
@@ -46,7 +46,7 @@ function noir (serializers, keys, censor) {
   return shape
 
   function redact () { return {toJSON: mask} }
-  function mask () { return censor }
+  function mask (key) { return typeof censor === 'function' ? censor(this[key]) : censor }
```

Comment:
can we make sure this logic path had a unit test

Reply:
Oh sure. I wasn't done with it. :)